.github/conversational/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+**Note: for support questions, please use stackoverflow**. This repository's issues are reserved for feature requests and bug reports.
```

Comment:
The communication channels are Discord and mailing list. 
Please edit thoroughly before you procure information from other sources. Just like the proverb in Hindi. :laughing: